manuscript/chapter3.txt

```diff
@@ -7,29 +7,29 @@ C> By [John Haynes](https://www.youtube.com/channel/UCJVXolxwB4x3EsBAzSACCTg) |
 
 ## Useful Software Tools
 
-There is no reason to reinvent the wheel as in most situations someone else has already created a tool that will perform the task needed. The same is true for using software to assist in cracking passwords. The general workflow for cracking a password is hash extraction, hash identification, attacking the hash with general methods, and attacking the hash with custom methods. Tools that can assist in these phases are [Mimikatz](https://github.com/gentilkiwi/mimikatz), [Hashcat](https://hashcat.net/hashcat/), [John the Ripper](https://github.com/openwall/john), [Passware](https://www.passware.com/), [Gov Crack](https://www.govcrack.com/), custom scripts often shared on GitHub and many more. Some tools like Passware are paid tools, and while there is nothing wrong with a paid tool, this paper will focus on using the free tool called Hashcat. Gov Crack has a graphical user interface (GUI) while Hashcat and John the Ripper use command-line interfaces (CLI). Normally GUI interfaces allow for ease of access but tend to lack the flexibility of CLI tools. Nearly all of the custom scripts that are used for hash extraction and are posted on GitHub are going to be CLI-based tools. If the reader is unfamiliar with the command line, that should not be a limiting factor for at least understanding the methods discussed in this paper and there will be step-by-step instructions on how to crack a password hash in Hashcat. The focus on a particular set of tools over another is due to personal experience with certain tools and no bias towards any particular tool is intended as many tools can do the same thing and overlap with each other with certain functions.
+Don't reinvent the wheel! There are a number of password cracking tools out there, both free and paid. This chapter will use [Hashcat](https://hashcat.net/hashcat/), a free and open-source tool with a command-line interface (CLI). Though they may be intimidating, CLI tools are frequently more flexible than graphical user interface (GUI) tools. Whether you use Hashcat or another tool, such as [Mimikatz](https://github.com/gentilkiwi/mimikatz), [John the Ripper](https://github.com/openwall/john), [Passware](https://www.passware.com/), [Gov Crack](https://www.govcrack.com/), or a custom script from GitHub, the general workflow is the same. The tool will help you extract and identify the hash and attack it with general methods followed by custom methods if the general ones fail. The right tool to choose is the one that works for your purposes. Experiment and see what you like best.
```

Comment:
I like how you condensed it, but it lost some of the nuances I was trying to communicate in the longer version. The condensed version implies that Mimikatz can crack passwords, but it is only used for hash extraction. John the Ripper can be used for hash extraction and cracking. Passware can crack a password without extracting the hash as is needed for Varacrypt/Truecrypt. It looks like Gov Crack has changed its website since writing this so it can be removed altogether. I'll leave it up to you to figure out how you want to communicate that.